bundles/org.eclipse.build.tools/src/org/eclipse/releng/generators/TestResultsGenerator.java

```diff
@@ -945,7 +945,7 @@ private void writeHTMLResultsTable(ArrayList<String> foundConfigs, ResultsTable
         htmlString = htmlString + EOL + "<h3 id=\"UnitTest\">Unit Test Results</h3>" + EOL;
 
         htmlString = htmlString
```

Comment:
use `getBuildType().equals("Y")` to check whether you are creating test results page for Y build. if yes point to https://ci.eclipse.org/releng/job/YPBuilds/ and in the else part point to https://ci.eclipse.org/releng/job/AutomatedTests/